### ISSUE TITLE
[MM 5.6.0] White page after using file upload dialog

### ISSUE THREAD

**Issue by somenet** [mattermost/mattermost-server#10011]
#### Summary
When users use the website's image upload, it uploads the file and then the page goes white.

After reloading the page (F5) the file is attached and the thumbnail generated, ready to be posted.

#### Steps to reproduce
1: click on attach file (file selection dialog opens)
![mm-screen-1](https://user-images.githubusercontent.com/5354356/50096989-2afa5e80-0219-11e9-86d7-f8e618ff6697.png)

2: select a gif or other image
![mm-screen-2](https://user-images.githubusercontent.com/5354356/50096991-2afa5e80-0219-11e9-8583-3886c690a7cd.png)

3: wait for upload to complete (Page whites out)
![mm-screen-3](https://user-images.githubusercontent.com/5354356/50096992-2afa5e80-0219-11e9-9010-a5d57329b3c2.png)

4: Press reload page
![mm-screen-4](https://user-images.githubusercontent.com/5354356/50096993-2b92f500-0219-11e9-8ce4-28eb47b0abba.png)

5: see uploaded file.
![mm-screen-5](https://user-images.githubusercontent.com/5354356/50096996-2b92f500-0219-11e9-9a5a-9f0dddd66294.png)

(6: write a comment and post)
![mm-screen-6](https://user-images.githubusercontent.com/5354356/50096998-2c2b8b80-0219-11e9-84ea-a09f65782b8c.png)

#### Expected behavior
I expect the page not to white out/crash.

#### Observed behavior (that appears unintentional) 
Page whites out.

**Comment by somenet** [comment]
Found some chromium logs
```
react-dom.production.min.js:171 TypeError: Cannot read property 'toFixed' of undefined
    at t.value (file_progress_preview.jsx:31)
    at Ka (react-dom.production.min.js:156)
    at Qa (react-dom.production.min.js:155)
    at $a (react-dom.production.min.js:162)
    at Oi (react-dom.production.min.js:192)
    at Ii (react-dom.production.min.js:193)
    at co (react-dom.production.min.js:205)
    at uo (react-dom.production.min.js:204)
    at Ui (react-dom.production.min.js:200)
    at Object.enqueueSetState (react-dom.production.min.js:130)
ai @ react-dom.production.min.js:171
n.callback @ react-dom.production.min.js:179
ra @ react-dom.production.min.js:125
na @ react-dom.production.min.js:125
lo @ react-dom.production.min.js:214
co @ react-dom.production.min.js:205
uo @ react-dom.production.min.js:204
Ui @ react-dom.production.min.js:200
enqueueSetState @ react-dom.production.min.js:130
g.setState @ react.production.min.js:13
(anonymous) @ create_post.jsx:622
(anonymous) @ file_upload.jsx:272
a.emit @ index.js:133
a @ client.js:714
XMLHttpRequest.send (async)
_._end @ client.js:772
_.end @ client.js:676
f @ file_upload.jsx:280
(anonymous) @ file_upload.jsx:251
(anonymous) @ file_upload.jsx:236
(anonymous) @ file_upload.jsx:334
(anonymous) @ react-dom.production.min.js:14
f @ react-dom.production.min.js:15
(anonymous) @ react-dom.production.min.js:15
A @ react-dom.production.min.js:17
O @ react-dom.production.min.js:18
P @ react-dom.production.min.js:18
C @ react-dom.production.min.js:18
L @ react-dom.production.min.js:21
Tn @ react-dom.production.min.js:84
po @ react-dom.production.min.js:216
Fe @ react-dom.production.min.js:39
On @ react-dom.production.min.js:86
react-dom.production.min.js:204 Uncaught TypeError: Cannot read property 'toFixed' of undefined
    at t.value (file_progress_preview.jsx:31)
    at Ka (react-dom.production.min.js:156)
    at Qa (react-dom.production.min.js:155)
    at $a (react-dom.production.min.js:162)
    at Oi (react-dom.production.min.js:192)
    at Ii (react-dom.production.min.js:193)
    at co (react-dom.production.min.js:205)
    at uo (react-dom.production.min.js:204)
    at Ui (react-dom.production.min.js:200)
    at Object.enqueueSetState (react-dom.production.min.js:130)
```

**Comment by jasonblais** [comment]
Thanks @somenet for the report. Just a few questions to help narrow down the issue.

1. Are you using the Desktop App, or Chrome browser?
2. Which operating system do you have, Windows, Mac, or Linux?
3. How many users does this reproduce on your system, do you know?

**Comment by jasonblais** [comment]
@somenet We weren't able to reproduce the issue, but have added a fix which we expect to resolve it.

The fix is included in 5.6.1 released yesterday. It is available to download at https://mattermost.com/download/

Closing the issue, but please feel welcome to reopen if you continue to see the issue after the upgrade.
